docs/diagrams/AddPointsSequenceDiagram.puml

```diff
@@ -0,0 +1,91 @@
+@startuml
+!include style.puml
+skinparam ArrowFontStyle plain
+
+box Logic LOGIC_COLOR_T1
+participant ":LogicManager" as LogicManager LOGIC_COLOR
+participant ":AddressBookParser" as AddressBookParser LOGIC_COLOR
+participant ":AddPointsCommandParser" as AddPointsCommandParser LOGIC_COLOR
+participant "p:Points" as Points LOGIC_COLOR
+participant "a:AddPointsCommand" as AddPointsCommand LOGIC_COLOR
+participant "r:CommandResult" as CommandResult LOGIC_COLOR
+participant "alice:Person" as Person LOGIC_COLOR
+end box
+
+box Model MODEL_COLOR_T1
+participant "m:Model" as Model MODEL_COLOR
+end box
+
+[-> LogicManager : execute(\n"addpoints \nn/Alice \np/50")
+activate LogicManager
+
+LogicManager -> AddressBookParser : parseCommand(\n"addpoints \nn/Alice \np/50")
+activate AddressBookParser
+
+create AddPointsCommandParser
+AddressBookParser -> AddPointsCommandParser
+activate AddPointsCommandParser
+
+AddPointsCommandParser --> AddressBookParser
+deactivate AddPointsCommandParser
+
+AddressBookParser -> DeleteCommandParser : parse(\n"n/Alice \np/50")
+activate AddPointsCommandParser
```

Comment:
I think `DeleteCommandParser` should be `AddPointsCommandParser` here

Reply:
Thanks for pointing out. Edited!